src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -95,6 +97,36 @@
     return future;
   }
 
+  /**
+   * This method is used for all internal calls to fetch PO lines without or with
+   * queries that search/filter on fields present in po_line
+   *
+   * @param limit
+   * @param offset
+   * @param query
+   * @return Completable future which holds {@link PoLineCollection}
+   */
+  CompletableFuture<PoLineCollection> getPoLines(int limit, int offset, String query) {
```

Comment:
Looks like this method is not in use so probably can be removed...

Reply:
This method is being called from other helpers to fetch the po lines without querying the view